ml-agents/setup.py

```diff
@@ -64,7 +64,7 @@ def run(self):
         "protobuf>=3.6",
         "pyyaml>=3.1.0",
         # Windows ver. of PyTorch doesn't work from PyPi
-        'torch>=1.6.0;platform_system!="Windows"',
+        'torch>=1.6.0,<1.8.0;platform_system!="Windows"',
```

Comment:
Not directly related to this change, but can we put a link to the windows instructions (either to our docs or external docs) as a comment here?